CHANGELOG.md

```diff
@@ -1,6 +1,16 @@
 # Change Log
 
 
+## [3.4.2](https://github.com/IBM/ignition/tree/3.4.2) (2022-11-21)
+[Full Changelog](https://github.com/IBM/ignition/compare/3.4.1...3.4.2)
+
+
+**Implemented enhancements:**
+
+- add error handling in netconf driver  [\#152](https://github.com/IBM/ignition/issues/152)
```

Comment:
You may need to change the description of GitHub issue 151 as it does not seem correct.

Reply:
Have made the requested changes